noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_intrinsic.rs

```diff
@@ -78,9 +78,11 @@ impl<F: AcirField + DebugToString, Registers: RegisterAllocator> BrilligContext<
 
         let heap_array = self.codegen_brillig_array_to_heap_array(target_array);
 
+        let radix_var = self.make_constant_instruction(F::from(radix as u128), 32);
```

Comment:
If you can deallocate the radix var after using it, so we don't leak the register